test/parse/test_entity_declaration.rb

```diff
@@ -18,25 +22,26 @@ def parse(internal_subset)
       REXML::Document.new(xml(internal_subset)).doctype
     end
 
+    public
     def test_empty
       exception = assert_raise(REXML::ParseException) do
         parse(<<-INTERNAL_SUBSET)
 <!ENTITY>
         INTERNAL_SUBSET
       end
       assert_equal(<<-DETAIL.chomp, exception.to_s)
-Malformed notation declaration: name is missing
+Malformed entity declaration
 Line: 5
-Position: 72
+Position: 70
 Last 80 unconsumed characters:
- <!ENTITY>  ]> <r/>
+>  ]> <r/> 
```

Comment:
[nits] There is a unnecessary white space.
```suggestion
>  ]> <r/>
```

Reply:
Unfortunately, the exception message contains trailing space. 
So I cannot remove it.

Reply:
Ahh I see. I'm so sorry mm